rasa/core/policies/ted_policy.py

```diff
@@ -721,19 +722,22 @@ def train(
 
         return self._resource
 
-    def _featurize_tracker_for_e2e(
+    def _featurize_tracker(
         self,
         tracker: DialogueStateTracker,
         domain: Domain,
-        interpreter: NaturalLanguageInterpreter,
+        precomputations: Optional[CoreFeaturizationPrecomputations] = None,
```

Comment:
As mentioned before, the precomputations must be messages and not just features which is why I went with the more generic name

Reply:
how about `featurized_messages` then? 🤔

Reply:
`tokenized_and_featurized_message` ... :D  Long term there could also be classifications in there from more complex recipes? 🤔  And it is not like all features end up in that dictionary because the `SingleStateFeaturizer` still creates all the multi-hot like features and creates sentence features from sequence features and so on

Reply:
okay, makes sense 👍🏻 Should we still stress that this is specific for end-to-end? We can still rename this once it's no longer specific to end-to-end

Reply:
That's true. But in that case we can simply rename this component, no?